### DISCUSSION TITLE
Where does the demo's date come from?

### DISCUSSION THREAD

**Discussion by N1N52** [akiver/cs-demo-manager#847] [Q&A] [answered]
Sorry if it was asked before, i use a lot Demo Manager and want to know these two things:

- In 3.5.2 version (Since CS2) date of the demo is the date of the download of the file, i remember that in csgo in the versions of demo manager, the date was the actual date of the game, is this correct? is there a way to order by the actual date of the game instead of the date of the download/creation of the file?

- The other thing is mirv_deathmsg and those commands to edit fragmovies, do you know if they will be implemented in futher versions?? is a must feature for all of us who want to do our own creations.

Thanks for the time. Cheers.

**Comment by akiver** [comment, marked as answer]
1. It's the same as in V2 - the real date is detected only for MM demos that have its `.info` file next to the `.dem` file (the date is stored in the `.info` file)
2. No idea, you can ask it on the HLAE repo https://github.com/advancedfx/advancedfx or on the HLAE discord
